pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -598,7 +599,7 @@ func (o *StartBuildOptions) RunStartBuildWebHook() error {
 		config, err := o.ClientConfig.ClientConfig()
 		if err == nil {
 			if url, _, err := restclient.DefaultServerURL(config.Host, "", unversioned.GroupVersion{}, true); err == nil {
-				if url.Host == hook.Host && url.Scheme == hook.Scheme {
+				if netutil.CanonicalAddr(url) == netutil.CanonicalAddr(hook) && url.Scheme == hook.Scheme {
```

Comment:
So the only other place I found was here:
https://github.com/openshift/origin/blob/master/pkg/cmd/cli/cmd/login/helpers.go#L28
https://github.com/openshift/origin/blob/master/pkg/cmd/cli/cmd/login/helpers.go#L40

However, I think the config is stored with the canonical address, so it may not be necessary to fix.
@fabianofranz do we have to worry about using the canonical address there?